apps/web/src/server/applications/create-new-case/applicant/__tests__/applications-create-applicant.test.js

```diff
@@ -197,7 +196,6 @@ describe('applications create applicant', () => {
 					const element = parseHtml(response.text);
 
 					expect(element.innerHTML).toMatchSnapshot();
-					expect(element.innerHTML).toContain('error-summary-title');
```

Comment:
Applications team to review - after updating `govuk-frontend`, this assertion now receives HTML instead of the expected string. Is this correct?

Reply:
this is fixed now